Sources/keeta-secure-storage/SecureStorage.swift

```diff
@@ -43,4 +43,8 @@ public final class SecureStorage {
     public func removeValue(for key: String) {
         keychain.delete(key)
     }
+    
+    public func clear() {
+        (keychain as? KeychainSwift)?.clear()
```

Comment:
I would add the `clear` function to the `Keychain` protocol

Reply:
🤦 I thought that was a library type... will do